osquery/tables/system/windows/deviceguard_status.cpp

```diff
@@ -42,18 +42,18 @@ QueryData genHVCIStatus(QueryContext& context) {
     long vbs_status;
     data.GetLong("VirtualizationBasedSecurityStatus", vbs_status);
     r["vbs_status"] =
-        vbs_methods.size() < vbs_status ? vbs_methods[vbs_status] : "UNKNOWN";
+        vbs_methods.size() > vbs_status ? vbs_methods[vbs_status] : "UNKNOWN";
 
     long code_policy_status;
     data.GetLong("CodeIntegrityPolicyEnforcementStatus", code_policy_status);
     r["code_integrity_policy_enforcement_status"] =
-        enforcement_methods.size() < code_policy_status
+        enforcement_methods.size() > code_policy_status
             ? enforcement_methods[vbs_status]
```

Comment:
I think this should also be changed:
```suggestion
            ? enforcement_methods[code_policy_status]
```

Reply:
Thanks for catching. I fixed this elsewhere but forgot to include it in the PR.